modules/backend/assets/js/october.simplelist.js

```diff
@@ -15,10 +15,11 @@
     var SimpleList = function (element, options) {
 
         var $el = this.$el = $(element)
+        var is_mobile = /Mobi/.test(navigator.userAgent);
```

Comment:
Note that the HTML element in the backend is populated with a number of "feature flag classes", so you could just detect if that element has a .mobile class on it, although I'm uncertain what is being tested to trigger that